docs/user-guide/index.rst

```diff
@@ -2,8 +2,24 @@
 
 .. include:: ModelAlignment.rsts
 
+:ref:`Prerequisite Obtaining a Pre-Trained Model <prerequisite>`
+   This section provides instructions on how to download pre-trained LLMs in .nemo format. The following section will use These base LLMs for further fine-tuning and alignment. 
+
+:ref:`Model Alignment by Supervised Fine-Tuning (SFT) <sft>`
+   In this section, we walk you through the most straightforward alignment method, using a supervised dataset in the prompt-response pairs format, to fine-tune the base model to the desired behavior.
+
+:ref:`Model Alignment by RLHF <model-aligner-sft>`
+   RLHF is the next step up in alignment and is still responsible for most state-of-the-art chat models. In this section, we walk you through the process of RLHF alignment, including training a reward model and the RLHF training with the  PPO algorithm.
+ 
+:ref:`Model Alignment by SteerLM Method <model-aligner-steerlm>`
+   SteerLM is a novel approach developed by the NVIDIA. SteerLM simplifies alignment compared to RLHF. It is based on SFT but allows user-steerable AI by enabling you to adjust attributes at inference time.
+
+:ref:`Model Alignment by Direct Preference Optimisation (DPO) <model-aligner-dpo>`
+   DPO is a simpler alignment method compared to RLHF. DPO introduces a novel parameterization of the reward model in RLHF. This parameterization allows us to extract the corresponding optimal 
+
 .. toctree::
    :maxdepth: 4
+   :titlesonly:
 
    SFT.rst
```

Comment:
do we not need to update the SFT.rst file?

Reply:
The SFT.rst file already had anchor links in the appropriate places.

https://github.com/NVIDIA/NeMo-Aligner/blob/main/docs/user-guide/SFT.rst?plain=1#L3
https://github.com/NVIDIA/NeMo-Aligner/blob/main/docs/user-guide/SFT.rst?plain=1#L49